### ISSUE TITLE
Sybase BigTime is appending some default date along with time when using Default Dialect

### ISSUE THREAD

**Issue by muskaan62** [jOOQ/jOOQ#17252]
### Your question

when using default dialect to retrieve BigTime data from Sybase is date is appending infornt of time using jconn driver.
```
Connection conn = DriverManager.getConnection(url, user, password);
            Statement st = conn.createStatement();
            DSLContext context = DSL.using(conn, SQLDialect.DEFAULT);
            Field<?> binary_col = DSL.field("BigTime_col", SQLDataType.VARCHAR);
            Map<Field<?>, Object> fieldMap = new LinkedHashMap<>();
            fieldMap.put(binary_col, "10:02:12.438");
            System.out.println(fieldMap);
            Query insertQueryTemplate = context.insertInto(DSL.table("\"dbo\".\"DateTypes\""),
                    fieldMap.keySet()).values(fieldMap.values());
            Batch batch = context.batch(insertQueryTemplate);
            batch.execute();
            String query = "select * from \"dbo\".\"DateTypes\"";
            ResultQuery resultQuery = context.resultQuery(query);
            System.out.println(resultQuery.fetchOne());

```
![image](https://github.com/user-attachments/assets/b5adc64f-e546-48a4-a649-fe114ac20653)

### jOOQ Version

3.17.3

### Database product and version

sybase ASE

### Java Version

11

**Comment by lukaseder** [comment]
Thanks for your message. Why are you using the `SQLDialect.DEFAULT` with Sybase?

**Comment by muskaan62** [comment]
hey @lukaseder  because ASE dialect has unsigned type issues and fix will be available in 3.20 so for that i am using DEFAULT here.

**Comment by lukaseder** [comment]
You cannot expect `DEFAULT` to work here! You have these 2 options:

- Work with 3.20.0-SNAPSHOT
- Patch your 3.17 version as instructed here: https://github.com/jOOQ/jOOQ/issues/17195#issuecomment-2348239314

**Comment by lukaseder** [comment]
I'm assuming that nothing else needs to be done here. `SQLDialect.DEFAULT` can't be expected to work with Sybase ASE (or with any RDBMS for that matter), and #17195 addressed the actual issue for jOOQ 3.20.0-SNAPSHOT
